receiver/zookeeperreceiver/metadata.yaml

```diff
@@ -1,5 +1,11 @@
 type: zookeeperreceiver
```

Comment:
Should we take the opportunity to rename this to `zookeeper` so we can use `metadata.Type` instead of `typeStr`?

Reply:
You're right, I missed that when searching for `metadata.Type` usages. I agree, let's wait to make that change.